examples/e2e/maas/resource_cluster.tf

```diff
@@ -0,0 +1,63 @@
+
+resource "spectrocloud_cluster_maas" "cluster" {
+  name               = "maas-picard-cluster"
+  cluster_profile_id = spectrocloud_cluster_profile.profile.id
+  cloud_account_id   = data.spectrocloud_cloudaccount_maas.account.id
+
+  cloud_config {
+    domain = var.maas_domain # "maas.sc"
+  }
+
+  # To override or specify values for a cluster:
+
+  # pack {
```

Comment:
```
  cluster_profile {
    id = local.profile_ids[each.value.profiles.infra]

#    pack {
#      name   = "spectro-rbac"
#      tag    = "1.0.0"
#      values = <<-EOT
#        charts:
#          spectro-rbac:
#            ${indent(4, replace(yamlencode(each.value.rbac), "/((?:^|\n)[\\s-]*)\"([\\w-]+)\":/", "$1$2:"))}
#      EOT
#    }
  }
```
Using `cluster_profile` with commented pack values in example as `pack` is deprecated